lib/spack/spack/cmd/install.py

```diff
@@ -160,8 +160,15 @@ def setup_parser(subparser):
         action='store_true',
         help="Show usage instructions for CDash reporting"
     )
+    subparser.add_argument(
+        '-y', '--yes-to-all',
+        action='store_true',
+        dest='yes_to_all',
+        help="""assume "yes" is the answer to every confirmation request
+to run completely non-interactively, also specify--no-checksum."""
```

Comment:
Can you make this a separate sentence, so that it's clearer to the user that it's a separate statement from the previous?